azure/keyvault/variables.tf

```
@@ -32,3 +32,9 @@ variable "readers" {
   type        = list(string)
   description = "Define a Azure Key Vault access policy"
 }
+
+variable "isProtect" {
```

Looking at all terraform modules we've made so far, we've used the same convention as terraform: instead of **camelCase**, we've used **underscore**. So, it might be a good idea to keep the same pattern.

Also, the variable "isProtect" name doesn't seem to descriptive enough of what it does. I know exactly what it means because I know what the PR is for, but let's consider  someone that will read this code. The person will need to visit take a look at the module code to understand what this variable does.


```suggestion
variable "purge_protection_enabled" { 
```

---

I agree with the recommended  change

---

@filipefigueredo variable name must by clear to the user of the module, as they must reasoning around it when using. Saying that, I think just using `is_protected` is more lean and clear, and the property description can bring more detailed information about azure concept behind it.

Answered after a glass or two of 🍷